nilearn/datasets/func.py

```diff
@@ -98,6 +98,14 @@ def fetch_haxby(data_dir=None, n_subjects=1, fetch_stimuli=False,
     n_subjects: int, optional
         Number of subjects, from 1 to 6.
 
+        NOTE: n_subjects is deprecated from 0.2.5.1 and will be removed in 0.3
+        Use `subjects` instead.
+
+    subjects : int or list, optional
+        Number of subjects or specific list of subjects to load. If None is
+        provided all 6 subjects (maximum) will be loaded. By default, 2nd
+        subject will be loaded.
```

Comment:
If I put `subjects=None`, I will get all the subjects but, to get no data at all, I have to put `subjects=[]`.